asserts/gpgkeypairmgr.go

```diff
@@ -264,24 +264,27 @@ func (gkm *GPGKeypairManager) findByName(name string) (*gpgKeypairInfo, error) {
 var generateTemplate = `
 Key-Type: RSA
 Key-Length: 4096
-Subkey-Type: RSA
-Subkey-Length: 4096
 Name-Real: %s
-Creation-Date: %s
+Creation-Date: seconds=%d
 Preferences: SHA512
 `
 
+func (gkm *GPGKeypairManager) getGenerateParameters(passphrase string, name string) string {
```

Comment:
generateParameters or parametersForGenerate  or also contractions of those would be ok,  get* is not used accessors in go, this one is not an accessor as such but still reads too much as one